contents/dataVis.md

```diff
@@ -0,0 +1,390 @@
+    current_figure()
+    filename = "customPlotLeg" # hide 
+    Options(current_figure(); filename, caption="custom plot legend", label="customplotlegend") # hide
+    """
+sco(s)
+```
+
+However, having to writte this so much code just for two lines can become cumbersome
+and tired, so if you plan on doing a lot of plots with the same general aesthetics
+then setting a theme will be better. We can do this with `set_theme!()` as the following
+example illustrates, not a particular good set of attributes but you get the idea. 
+
+```jl
+s = """
+    set_theme!(resolution = (600,400), backgroundcolor = (:orange, 0.5), 
+        fontsize = 16, font = "sans",
+        Axis = (backgroundcolor = :white, xgridstyle=:dash, ygridstyle=:dash),
+        Legend = (bgcolor = (:red,0.2), framecolor = :dodgerblue)
+        )
+
+    lines(1:10, (1:10).^2, label = "x²", linewidth = 2, linestyle = nothing,
+        axis = (; xlabel = "x", title = "title"))
+    scatterlines!(1:10, (10:-1:1).^2, label = "Reverse(x)²")
+    axislegend("legend", position = :ct)
+    current_figure()
+    set_theme!() # in order to go back to the default settings. 
+    filename = "setTheme" # hide 
+    Options(current_figure(); filename, caption="set theme", label="setTheme") # hide
+    """
+sco(s)
+```
+
+For more on `themes` please go to section xxx (Refs)
+
+Before moving on into the next section, it's worthwhile to see an example 
+where an `array` of attributes are passed at once to a plotting function. 
+For this example we will use the `scatter` plotting function to do a bubble plot. 
+
+```jl
+s = """
+    let
+        Random.seed!(123)
+        n = 100
+        fig, ax, pltobj = CairoMakie.scatter(randn(n), randn(n), color = randn(n), 
+            label = "Bubbles",colormap = :plasma, markersize = 25*rand(n), 
+            figure = (; resolution = (550,400)), axis = (; aspect= DataAspect()))
+        limits!(-3, 3, -3, 3)
+        Legend(fig[1,2], ax, valign = :top)
+        Colorbar(fig[1,2], pltobj, height = Relative(3/4))
+        fig
+        Options(current_figure(); filename ="bubbleplot", caption="Bubble plot", label="bubbleplot") # hide 
+    end
+    """
+sco(s)
+```
+
+Where we have decomposed the tuple `FigureAxisPlot` into `fig, ax, pltobj`, in order to 
+be able to add a `Legend` and `Colorbar` outside of the plot object. We will discuss 
+this, `layouts`,  in more detail in the [Layouts] section.
+
+Well, we have done some basic but still interesting examples to show
+how to use `Makie` and by now you might be wondering, what else can 
+we do? What are all the possible plotting functions available in `Makie`?
+To answer this question a `cheat sheet` is shown in [@fig:cheat_sheet_cairomakie]. These work
+specially well with `CairoMakie`. 
+
+![Plotting functions: Cheat Sheet. Output given by Cairomakie.](images/makiePlottingFunctionsHide.png){#fig:cheat_sheet_cairomakie}
+
+For completeness in [@fig:cheat_sheet_glmakie] we 
```

Comment:
```suggestion
For completeness, in [@fig:cheat_sheet_glmakie] we 
```
Or, did you mean "For completeness in [@fig:cheat_sheet_glmakie], we show [...]". I'm not sure.